### ISSUE TITLE
scatter symlog doesn't work properly

### ISSUE THREAD

**Issue by ianhbell** [matplotlib/matplotlib#15077]
### Bug report

**Bug summary**

When plotting scatter data with symlog y axis, the data in the linear part are not plotted

```python
import matplotlib.pyplot as plt
import numpy as np
x = np.linspace(0,1, 1000)
y = (np.random.random((len(x),))-0.5)*100
c = (np.random.random((len(x),))-0.5)*10
plt.scatter(x,y,c=c,marker='o',lw=0)
plt.yscale('symlog',linthreshy=10)
plt.show()
```

**Actual outcome**

![image](https://user-images.githubusercontent.com/11966765/63227234-9a0dc580-c1a1-11e9-9c14-c1177ff9c071.png)

**Matplotlib version**

  * Operating system:
  * Matplotlib version: 3.1.0
  * Matplotlib backend (`print(matplotlib.get_backend())`): Qt5Agg
  * Python version: 3.7

installed via conda, default channel

**Comment by timhoffm** [comment]
Thanks for the report. This does work as of master:

![grafik](https://user-images.githubusercontent.com/2836374/63548638-f02a9200-c52e-11e9-982f-cafee616b4b9.png)

There has been some modification concerning symlog recently. I'm assuming one of these fixed this implicitly.

**Comment by tacaswell** [comment]
Does this work on 3.1.x?

**Comment by tacaswell** [comment]
:sheep: Duplicate of https://github.com/matplotlib/matplotlib/issues/14265 which is fixed in 3.1.1

**Comment by ianhbell** [comment]
Thanks for the info, confirmed fixed in 3.1.1.  Sorry for the noise.

**Comment by tacaswell** [comment]
Thank you for reporting.  Despite the noise I would rather get multiple reports of critical regressions like this than no reports!
